Xamarin.Essentials/AppInfo/AppInfo.android.cs

```diff
@@ -48,5 +49,15 @@ static void PlatformShowSettingsUI()
             settingsIntent.AddFlags(ActivityFlags.ExcludeFromRecents);
             context.StartActivity(settingsIntent);
         }
+
+        static AppTheme PlatformRequestedTheme()
+        {
+            return (Platform.AppContext.Resources.Configuration.UiMode & UiMode.NightMask) switch
```

Comment:
Ensure the device is running Android Froyo or higher because UIMode was added in Android Froyo, API 8.0
if (Build.VERSION.SdkInt >= BuildVersionCodes.Froyo)

Reply:
Xamarin.Essentials only supports KitKat and above, so no need to check it was my thinking